plutus-core/untyped-plutus-core/testlib/Evaluation/Builtins/Integer/Common.hs

```diff
@@ -0,0 +1,83 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE TypeApplications    #-}
+{-# LANGUAGE TypeOperators       #-}
+module Evaluation.Builtins.Integer.Common
+where
+
+import Prelude hiding (and, not, or)
+
+import PlutusCore qualified as PLC
+import PlutusCore.MkPlc (builtin, mkIterAppNoAnn, mkTyBuiltin, tyInst)
+
+import Evaluation.Builtins.Common
+
```

Comment:
More utility functions for creating PLC terms.  I'm probably reinventing the wheel yet again here.